docs/5.0/config-reference.md

```diff
@@ -370,4 +352,23 @@ app_service:
      - name: "os"
        command: ["/usr/bin/uname"]
        period: "5s"
+
+## This section confitures the 'kubernetes service'
+kubernetes_service:
+    enabled: yes
+    # Optional Public & Listen Addr: Set these if you are connecting to 
+    # Teleport running inside a Kubernetes cluster. 
+    # Optional Public Addr
+    public_addr: [k8s.example.com:3026]
+    # Optional Listen Addr 
+    listen_addr: 0.0.0.0:3026
```

Comment:
I think this has to be different to the public address as it's used for the internal connections rather than external? @awly 

```suggestion
    listen_addr: 0.0.0.0:3027
```

Reply:
Added suggestion in another commit